dashboard/signals.py

```diff
@@ -13,12 +14,12 @@ def handle_create_programenrollment(sender, instance, created, **kwargs):  # pyl
     """
     When a ProgramEnrollment model is created/updated, update index.
     """
-    index_program_enrolled_users.delay([instance])
+    transaction.on_commit(lambda: index_program_enrolled_users.delay([instance.id]))
```

Comment:
I didn't either, I just happened upon it because the tests run in transactions so the tests were failing when looking up by id numbers